### PR TITLE
msm8939: Add Lenovo Vibe K5 Plus

### PR DESCRIPTION
**Pull request by ruizlenato**
image

### CODE REVIEW

File: dts/msm8916/msm8939-qrd-wt82918.dts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        model = "Lenovo Vibe K5 Plus (Wingtech WT82918)";
+        compatible = "wingtech,wt82918", "qcom,msm8939", "lk2nd,device";
+
+        panel {
+                compatible = "wingtech,wt82918-panel";
+
+                qcom,mdss_dsi_otm1901a_tm_1080p_video {
+                        compatible = "wingtech,otm1901a-tm";
+                };
+		qcom,mdss_dsi_boe_nt35595_1080p_video {
```
Comment (TravMurav) [inline dts/msm8916/msm8939-qrd-wt82918.dts:20]:
```suggestion
                };

		qcom,mdss_dsi_boe_nt35595_1080p_video {
```

Please keep a newline between nodes. Also please note that the dts files are intented with 8-wide hard tab, not spaces.

Comment (ruizlenato) [reply dts/msm8916/msm8939-qrd-wt82918.dts]: Done

**Review by TravMurav — COMMENTED** [review]
This looks good to me but there is still a mix up of tabs and spaces for the indentation. We use hard tabs in lk2nd. I'd appreciate if you could (auto-)format the dts file.

**Comment by TravMurav** [comment]
Thanks!

**Pull request merged**